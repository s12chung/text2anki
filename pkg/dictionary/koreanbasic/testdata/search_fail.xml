<!-- DO NOT EDIT. Generated in koreanbasic_test.go -->








<channel>
	<title>한국어 기초사전 개발 지원(Open API) - 사전 검색</title>
	<link>https://krdict.korean.go.kr</link>
	<description>한국어 기초사전 개발 지원(Open API) - 사전 검색 결과</description>
	<lastBuildDate>20210723122317</lastBuildDate> 
	<total>0</total>
	<start>1</start>
	<num>10</num>
	
	
	
</channel>


