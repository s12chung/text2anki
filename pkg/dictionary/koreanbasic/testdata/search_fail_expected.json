{
  "Items": null,
  "Total": 0,
  "Start": 1,
  "Number": 10
}